pkgs/development/python-modules/phidget22/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+  libphidget22,
+  libphidget22extra,
+}:
+let
+  pname = "phidget22";
+  version = "1.22.20250422";
+  format = "wheel";
+in
+buildPythonPackage {
+  inherit pname version format;
+
+  src = fetchPypi {
+    inherit pname version format;
+    python = "py3";
+    dist = "py3";
+    sha256 = "sha256-qV7Jr1HPuvlp2fu/wgB8Qku1K/90fgzTY0uxbe/mFAk=";
+  };
+
+  propagatedBuildInputs = [
+    libphidget22
+    libphidget22extra
+  ];
+
+  pythonImportsCheck = [ "Phidget22" ];
+
+  meta = {
+    homepage = "https://www.phidgets.com/docs/Language_-_Python";
+    changelog = "https://phidgets.com/?view=changelog";
+    description = "Phidget22 is the official Python library for working with Phidgets in Python applications. It provides lightweight bindings to the native phidget22 library.";
```

Comment:
Ending `.` seems to conflict with this guide https://github.com/NixOS/nixpkgs/blob/0d91072511f0c5f012904211dfb75793b88523a1/pkgs/README.md?plain=1#L406C1-L413C33

I interpreted this as `Lightweight bindings to the native C library for Phidget devices`, but please feel free to rephrase if there's a more accurate description - just sharing my take.